kinisi/parser.py

```diff
@@ -225,3 +225,129 @@ def get_indices(
         indices = sc.Variable(dims=['atom'], values=indices)
         drift_indices = sc.Variable(dims=['atom'], values=drift_indices)
         return indices, drift_indices
+
+
+class MDAnalysisParser(Parser):
+    """
+    Parser for MDAnalysis structures.
+
+    Takes an MDAnalysis.Universe object as an input. 
+
+    :param universe: MDanalysis universe object to be parsed
+    :param specie: Specie to calculate diffusivity for as a String, e.g. :py:attr:`'Li'`.
+    :param time_step: The input simulation time step, i.e., the time step for the molecular dynamics integrator. Note, 
+        that this must be given as a :py:mod:`scipp`-type scalar. The unit used for the time_step, will be the unit 
+        that is use for the time interval values.
+    :param step_skip: Sampling freqency of the simulation trajectory, i.e., how many time steps exist between the
+        output of the positions in the trajectory. Similar to the :py:attr:`time_step`, this parameter must be
+        a :py:mod:`scipp` scalar. The units for this scalar should be dimensionless.
+    :param dt: Time intervals to calculate the displacements over. Optional, defaults to a :py:mod:`scipp` array
+        ranging from the smallest interval (i.e., time_step * step_skip) to the full simulation length, with 
+        a step size the same as the smallest interval.
+    :param dimension: Dimension/s to find the displacement along, this should be some subset of `'xyz'` indicating
+        the axes of interest. Optional, defaults to `'xyz'`.
+    :param distance_unit: The unit of distance used in the input structures. Optional, defaults to angstroms.
+    :param sub_sample_atoms: Subsample the atoms in the trajectory. Optional, defaults to 1.
+    :param sub_sample_traj: Subsample the trajectory. Optional, defaults to 1.
```

Comment:
Perhaps we should aim to have parity between the different parsers in terms of features. Pymatgen doesn't have these options. Thoughts?

Reply:
Agree r.e. parity, have updated